webbpsf/trending.py

```diff
@@ -45,6 +45,10 @@ def wavefront_time_series_plot(opdtable, start_date=None, end_date=None, label_v
         Start and end dates for the plot time range. Default is March 2022 to present.
     label_visits : bool
         Label program_id:visit_id for each WFS visit.
+    label_events : bool
+        Label events along the x-axis in time defined in the events {}
+    full_path : bool
+        Whether the fileName column already contains a full file path or not
```

Comment:
Question. We might be able to avoid needing this parameter by making the code simply check if the path in the OPD table is a full path, via `os.path.isfile()`. If the filename provided is a valid full path to an existing file, use it as-is. If not, try appending the webbpsf data path on the front end, etc, to make an absolute path.

Reply:
I like that I will switch to that implementation as I agree its much cleaner.